task_templates/README.md

```diff
@@ -1,50 +1,73 @@
+* transforms - meant to be used as a preprocessing step in a blueprint
+* pipelines - meant to be added as a single task blueprint, these have some preprocessing and an
+  estimator built in
 
-Our estimator examples are also different than the pipelines examples because, we decided to implement the score
-hooks for all the estimator templates, and for the pipelines templates we did not. You only don't have to use a score
-hook if your model inherits from sklearn, pytorch, keras, or xgboost and doesn't need any additional functionality
+Our estimator examples are also different than the pipelines examples because these templates have
+score hooks implemented. The pipelines examples do not have score hooks to demonstrate the automatic
+scoring functionality of the DRUM middleware layer.
 
-Below is an enumeration of the templates we provide
+If a template is tagged as "Verified", this means we have automated tests which guarantee
+integration functionality with the DataRobot platform
 
 ## Transforms
-* `custom_class_python` - Shows how to define your own transformation class without relying on external libraries
-* `python_missing_values` - Uses builtin pandas functionality to impute missing values
- * `python3_sklearn_transform_hyperparameters` - Verified
- * `python3_sklearn_transform` - Verified
-
-## Estimators 
-* `python_calibrator` - Shows how to create an estimator task for doing prediction calibration, which
-is usually done in DataRobot as an additional estimator step after the main estimator
-* `python_anomaly` - Shows how to use sklearn to create an anomaly estimator
-* `python_binary_classification` - Shows how to use sklearn to create an estimator for binary classification problems
-* `python_multiclass_classification` 
-* `python_regression`
-* `python_regression_with_custom_class` - Does not inherit sklearn for the estimator
-* `r_classification`
-* `r_regression`
+
+* `python_missing_values` - Good starter template. It uses builtin pandas functionality to impute
+  missing values
+* `custom_class_python` - Shows how to define your own transformation class without relying on
+  external libraries
+* `python3_sklearn_transform_hyperparameters` - Hyperparameters functionality is currently still in
+  development
+* `python3_sklearn_transform` - Verified - This is a python template implementing a
+  preprocessing-only SKLearn pipeline, handling categorical and numeric variables
+
+## Estimators
+
+* `python_anomaly` - Shows how to create an anomaly estimator
+* `python_binary_classification` - Shows how to create an estimator for binary
+  classification problems
+* `python_multiclass_classification` - Implements a linear classifier with SGD training
+* `python_regression` - Implements a linear regressor with SGD training
+* `python_regression_with_custom_class` - Uses a custom python class, CustomCalibrator, so that we
+  can store the complex state inside the object and then re-use it during scoring.
```

Comment:
so that we can store the complex state inside the object and then re-use it during scoring -> which implements fit and score from scratch